chaincfg/params.go

```diff
@@ -297,33 +297,7 @@ var MainNetParams = Params{
 	GenerateSupported:        false,
 
 	// Checkpoints ordered from oldest to newest.
-	Checkpoints: []Checkpoint{
```

Comment:
Just a question, maybe it would be better to create something like `BblTestnetParams` without any checkpoints instead of removing then from existing networks ?

Reply:
Yeah, I thought of that but I wanted to minimize changes. There are multiple files (at least 18) that make use of Testnet3Params and some of them check some conditions based on network names. Moreover, the different address formats (p2pkh, p2sh, ...) start with unique characters for each network. I suspected that changing them would cause problems for the btcwallet package. I skimmed through its code. Apparently btcwallet uses btcd to import network configs.

Reply:
okay so this seem like a good follow through task (maybe create a github issue ?) to investigate what packages are used by wallet and what we need to do to be compatible with when creating our own network params config.

Reply:
Yeah, I agree. I think we should probably start by forking the btcwallet and work on our own wallet (which can be called bblwallet).